### PR TITLE
VPA - Status e2e test

### PR DESCRIPTION
**Pull request by krzysied**
Adding e2e tests for updater. The tests cover the Updater behavior regarding the Admission Controller status availability.

ref #2738

### CODE REVIEW

File: vertical-pod-autoscaler/e2e/v1/updater.go [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+		stopCh := make(chan struct{})
+		statusUpdater := status.NewUpdater(
+			f.ClientSet,
+			status.AdmissionControllerStatusName,
+			status.AdmissionControllerStatusNamespace,
+			statusUpdateInterval,
+			"e2e test",
+		)
+		defer func() {
+			// We should clean up the Admission Controller status.
```
Comment (bskiba) [inline vertical-pod-autoscaler/e2e/v1/updater.go]: // Schedule clean up of Admission Controller status.

Comment (krzysied) [reply vertical-pod-autoscaler/e2e/v1/updater.go]: Done.

**Comment by bskiba** [comment]
/lgtm
/approve

Thanks for taking care of this issue end to end, including tests!

**Pull request merged**